traits/trait_handlers.py

```diff
@@ -419,6 +428,9 @@ def info(self):
 
         return result
 
+    def info_none(self):
```

Comment:
I think this method can be removed: `info_none` is only used in `ThisClass.__init__`.